test/test_torch.py

```diff
@@ -3965,16 +3965,17 @@ def test_gesv(self):
 
     @staticmethod
     def _test_gesv_batched(self, cast):
+        from test_autograd import random_fullrank_matrix_distinct_singular_value as fullrank
```

Comment:
Unfortunately, in our internal CI, an import of this form doesn't work (because the tests get put in a different point in the module hierarchy. Is there some other way we can easily organize this? For example, put this function in `common.py`?

Reply:
I could add them in `common.py` or create a new file `common_linalg.py` which will contain all linear algebra based factory functions in it. `random_fullrank_matrix_distinct_singular_values` is one among a bunch of similar-looking functions. Which one would you prefer?

Also, `_test_matrix_power` has the same import as well.

Reply:
i think in `common.py` is sufficient.